jkube-kit/jkube-kit-micronaut/src/main/java/org/eclipse/jkube/micronaut/enricher/MicronautHealthCheckEnricher.java

```diff
@@ -85,10 +85,15 @@ private Probe buildProbe(Integer initialDelaySeconds, Integer periodSeconds){
       return null;
     }
 
-    final String firstImagePort = getImages().stream().findFirst()
+    String port = getImages().stream().findFirst()
         .map(ImageConfiguration::getBuild).map(BuildConfiguration::getPorts)
         .orElse(Collections.emptyList()).stream()
         .findFirst().orElse(null);
+
+    if (port == null) {
+      port = extractPort(getMicronautConfiguration(getContext().getProject()), null);
```

Comment:
Do you think we can move this to `orElse` method of the above statement?

Reply:
Thanks! applied the changes.